modules/preparing-an-inital-cluster-deployment-for-mce-disconnected.adoc

```diff
@@ -0,0 +1,129 @@
+    skipTLS: true
+  mirror:
+    platform:
+      architectures:
+        - "amd64"
+      channels:
+        - name: stable-4.12 <4>
+          type: ocp
+    additionalImages:
+      - name: registry.redhat.io/ubi8/ubi:latest
+    operators:
+      - catalog: registry.redhat.io/redhat/redhat-operator-index:v4.12 <5>
+        packages: <6>
+          - name: multicluster-engine <7>
+          - name: local-storage-operator <8>
+----
+
++
+NOTE: This file is required by the `oc mirror` command when mirroring content.
+
++
+<1> Add `archiveSize` to set the maximum size, in GiB, of each file within the image set.
+<2> Set the back-end location to save the image set metadata to. This location can be a registry or local directory. It is required to specify `storageConfig` values, unless you are using the Technology Preview OCI feature.
+<3> Set the registry URL for the storage backend.
+<4> Set the channel to retrieve the {product-title} images from.
+<5> Set the Operator catalog to retrieve the {product-title} images from.
+<6> Specify only certain Operator packages and channels to include in the image set. Remove this field to retrieve all packages in the catalog.
+<7> The MCE Operator packages and channels.
+<8> The LSO Operator packages and channels.
+
+. To mirror the specific {product-title} image repository, the MCE Operator and Local Storage Operator(LSO), run the following command:
+
++
+[source,terminal]
+----
+oc mirror --dest-skip-tls --config ocp-mce-imageset.yaml docker://<your-local-registry-dns-name>:<your-local-registry-port>
+----
+
+
+. Update the registry and certificate in the `install-config.yaml` file:
+
++
+.Example `imageContentSources.yaml`
+
+[source,yaml]
+----
+  imageContentSources:
+    - source: "quay.io/openshift-release-dev/ocp-release"
+      mirrors:
+        - "<your-local-registry-dns-name>:<your-local-registry-port>/openshift/release-images"
+    - source: "quay.io/openshift-release-dev/ocp-v4.0-art-dev"
+      mirrors:
+        - "<your-local-registry-dns-name>:<your-local-registry-port>/openshift/release"
+    - source: "registry.redhat.io/ubi8"
+      mirrors:
+        - "<your-local-registry-dns-name>:<your-local-registry-port>/ubi8"
+    - source: "registry.redhat.io/multicluster-engine"
+      mirrors:
+        - "<your-local-registry-dns-name>:<your-local-registry-port>/multicluster-engine"
+    - source: "registry.redhat.io/rhel8"
+      mirrors:
+        - "<your-local-registry-dns-name>:<your-local-registry-port>/rhel8"
+    - source: "registry.redhat.io/redhat"
+      mirrors:
+        - "<your-local-registry-dns-name>:<your-local-registry-port>/redhat"
+----
+
++
+Additionally, ensure your certificate is present in the `additionalTrustBundle` field of the `install-config.yaml`.
+
++
+.Example `install-config.yaml`
+[source,yaml]
+----
+additionalTrustBundle: |
+  -----BEGIN CERTIFICATE-----
+  . zzzzzzzzzzz
+  .  -----END CERTIFICATE-----
```

Comment:
```suggestion
  -----BEGIN CERTIFICATE-----
  zzzzzzzzzzz
  -----END CERTIFICATE-----
```
I think these periods should not be in the sample certificate unless they were put there for a specific reason.

Reply:
correct , good catch